common/src/chain/config.rs

```diff
@@ -72,7 +74,7 @@ impl ChainConfig {
     }
 }
 
-const MAINNET_ADDRESS_PREFIX: &str = "mlt";
+const MAINNET_ADDRESS_PREFIX: &str = "mt";
```

Comment:
There's a conversation on slack about this. `mt` has a few issues when it comes to the testnet prefix

Reply:
Generally speaking I can understand the issue with `l` in an address but I wonder is it an actual issue as a prefix for *Mintlayer's* bech32 addresses. I'm looking at `mt` and thinking where did the `l` go. I don't feel strongly about this but I think it's extremely unlikely anybody would typo/read the prefix incorrectly.